glassfish-tck-runner/pom.template.xml

```diff
@@ -363,6 +363,42 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>web</id>
+            <!-- Pull in the additional TCK artefact and scan it -->
+            <dependencies>
+                <dependency>
+                    <groupId>jakarta.enterprise</groupId>
+                    <artifactId>cdi-tck-web-impl</artifactId>
+                    <version>${cdi.tck.version}</version>
+                    <scope>test</scope>
+                    <exclusions>
+                        <exclusion>
+                            <artifactId>container-se-api</artifactId>
+                            <groupId>org.jboss.arquillian.container</groupId>
+                        </exclusion>
+                    </exclusions>
+                </dependency>
+            </dependencies>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <suiteXmlFiles>
+                                <suiteXmlFile>src/test/tck20/tck-tests.xml</suiteXmlFile>
+                            </suiteXmlFiles>
+                            <excludedGroups>javaee-full,se</excludedGroups>
+                            <dependenciesToScan>
+                                <dependency>jakarta.enterprise:cdi-tck-core-impl</dependency>
+                                <dependency>jakarta.enterprise:cdi-tck-web-impl</dependency>
```

Comment:
I'm not sure this dependency exists anymore?
It's not bundled in the [actual TCK bundle](https://repo1.maven.org/maven2/jakarta/enterprise/cdi-tck-dist/4.1.0/cdi-tck-dist-4.1.0-dist.zip) and there's only a milestone version of it published in maven central

Reply:
@luiseufrasio don't forget to remove this line
```suggestion
```